datadog-setup.php

```diff
@@ -1630,6 +1645,35 @@ function ($line) {
     }
 }
 
+function map_env_to_ini($env)
+{
+    $setting = explode('_', $env, 3);
+    if ($setting[0] !== 'DD') {
+        return null;
+    }
+
+    $ini = 'datadog.';
+    switch ($setting[1]) {
```

Comment:
I suppose technically index 1 ought to have an isset guard as well.

Reply:
Done